daemon/volumes.go

```diff
@@ -205,6 +233,17 @@ func (daemon *Daemon) registerMountPoints(container *Container, hostConfig *runc
 		}
 
 		if len(bind.Name) > 0 && len(bind.Driver) > 0 {
+			runtime.LockOSThread()
+			//  defer reset the label
+			defer label.SetFileCreateLabel("")
+			defer runtime.UnlockOSThread()
```

Comment:
This defer is in a for-loop, and probably just needs to wrap the SetFileCreateLabel (below) anyway.